x/supplier/client/cli/tx_create_claim.go

```diff
@@ -0,0 +1,55 @@
+package cli
+
```

Comment:
```suggestion
	TODO(@bryanchriswhite): Add unit tests for the CLI command when implementing the business logic.
```

See #59 for an example